bin/notify-on-location-hierarchy-addition

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -euo pipefail
+
+: "${SLACK_TOKEN:?The SLACK_TOKEN environment variable is required.}"
```

Comment:
Alternatively, `SLACK_TOKEN` could be removed from all the `notify-*` scripts (except `notify-slack`, which checks for both `SLACK_TOKEN` and `SLACK_CHANNLES`.

Reply:
Hmm, I want to say that we've required `SLACK_TOKEN` in all of the `notify-*` scripts so that they can fail loudly early on instead of waiting for it to fail at the `notify-slack` command.

Reply:
Hmm but then we'd do some data processing just to throw it away for a missing env var... you're right, I'll instead add the check for `SLACK_CHANNELS` to all `notify-*` scripts.

Reply:
Ha we had nearly the same thought at the same time. I wrote my follow up response before seeing your comment. :see_no_evil: